loader/cJSON.h

```diff
@@ -25,7 +25,9 @@
 
 #pragma once
 
-#include "loader_common.h"
+#include <stdint.h>
+
+#include <vulkan/vulkan_core.h>
 
 #if defined(__cplusplus)
```

Comment:
Since you are modifying this library interface anyway might as well remove the `#if defined(__cplusplus)` logic since this is a C only library.

Reply:
I thought about that but didn't just because its a non-issue for the most part.